engine/release-notes.md

```diff
@@ -203,6 +203,59 @@ Ubuntu 14.04 "Trusty Tahr" [docker-ce-packaging#255](https://github.com/docker/d
 
 ## Older Docker Engine EE Release notes
 
+### 18.03.1-ee-4 
+2018-10-25
+
+> *** NOTE: **** If you're deploying UCP or DTR, use Docker EE Engine 17.06.
```

Comment:
EE 18.09 or 17.06 I guess (I think we have a snippet somewhere explaining that 18.03 is a standalone ("engine") release.

(probably has to be mentioned in the other patch release below as well)

Reply:
@thaJeztah fixed. Ok to merge?